src/main/scala/io/scalac/tezos/translator/routes/utils/ReCaptcha.scala

```diff
@@ -74,23 +74,33 @@ object ReCaptcha {
           Left(Error("Can't do request to verify captcha"))
       }
   }
-
+  protected def checkScore(score:Option[Float]): Either[ErrorResponse, Unit] = {
+    score match {
+      case (Some(number)) =>
+    if (number > 0.35)
+    ().asRight
+    else
+    (Error ("You are bot"), StatusCode.Unauthorized).asLeft
+      case _ => (Error ("Empty Score"), StatusCode.Unauthorized).asLeft
```

Comment:
What does it mean when in the response we get a `None` ? 
"Empty Score" does not seem informative enough. Please make it more user friendly

Reply:
Empty Score shows only when the response is success and score is missing(None). Perhaps "Google response was succes but doesn't have "Score", Probably you used recaptcha v2"